src/picterra/client.py

```diff
@@ -106,23 +123,18 @@ def upload_raster(self, filename, name, folder_id=None, captured_at=None):
             name (str): A human-readable name for this raster
             folder_id (optional, str): Id of the folder this raster
                 belongs to.
-            captured_at (optional, str): ISO-8601 date and time at which
-                this raster was captured.
+            captured_at (optional, str): ISO-8601 date and time at which this
+                raster was captured, YYYY-MM-DDThh:mm[:ss[.uuuuuu]][+HH:MM|-HH:MM|Z];
+                e.g. "2020-01-01T12:34:56.789Z"
 
         Returns:
             raster_id (str): The id of the uploaded raster
         """
-        data = {
-            'name': name
-        }
-        if folder_id is not None:
-            data.update({
-                'folder_id': folder_id
-            })
-        if captured_at is not None:
-            data.update({
-                'captured_at': captured_at
-            })
+        data = {'name': name}
+        if folder_id != '':
```

Comment:
Let's use None, its the right thing to do in Python to indicate a param is not set. So I would revert this to what we had before

Reply:
Checked